src/api/app/views/webui2/webui/package/revisions.html.haml

```diff
@@ -0,0 +1,20 @@
+- @pagetitle = "Revision Log of #{@package}"
+- package_bread_crumb 'Revisions'
+
+.card
+  = render partial: 'tabs'
+
+  .card-body
+    - if @revisions
+      %h3 #{@pagetitle} (#{@lastrev})
+      .list-group
+        - @revisions.each do |rev|
+          .list-group-item{ id: "commit_item_#{rev}" }
+            = render(partial: 'commit_item', locals: { rev: rev })
+      - unless params[:showall]
+        %p
+          = link_to('Show all', action: 'revisions', project: @project, package: @package, showall: 1)
```

Comment:
This is also something that could use some :green_heart: We already have `kaminari`...